subsys/net/l2/ethernet/gptp/gptp_md.c

```diff
@@ -38,36 +38,13 @@ static void gptp_md_follow_up_prepare(struct net_pkt *pkt,
 	hdr = GPTP_HDR(pkt);
```

Comment:
The `net: gptp: fix follow_up timestamp and announce byte order` contains three fixes. Could you separate them to three commits?
Please also enhance the commit messages so that they answer the question "Why this change is needed". The current changes look somewhat ok but it would be nice to have a bit more information (for example links to spec etc would be really helpful if possible).

Reply:
Heys [jukkar](https://github.com/jukkar),
Already separated them to three commits, and add some reference in commit.
The document can be found at https://1.ieee802.org/wp-content/uploads/2019/03/802-1AS-rev-d8-0.pdf